### PR TITLE
Adds Intelligence Pack functions

### PR DESCRIPTION
**Pull request by chfunk**
Adds functions to enable, disable, and list intelligence packs

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@independentwork__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (chfunk). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

**Comment by markcowl** [comment]
@independentwork Failing tests.  Please fix.

File: src/ResourceManager/Common/Commands.ResourceManager.Common/Commands.ResourceManager.Common.csproj [hunk 1]
```diff
@@ -68,6 +68,10 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Microsoft.Azure.Common.2.1.0\lib\net45\Microsoft.Azure.Common.NetFramework.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Azure.Management.OperationalInsights, Version=0.9.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (markcowl) [inline src/ResourceManager/Common/Commands.ResourceManager.Common/Commands.ResourceManager.Common.csproj]: Why is this added here?  This adds an OperationInsights dependency to all AzureRM projects - meaning evry time it's updated, every package would have to be republished.  Please add this only to your project

File: src/ResourceManager/Common/Commands.ResourceManager.Common/packages.config [hunk 2]
```diff
@@ -6,6 +6,7 @@
   <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Common.Authentication" version="1.4.0-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.OperationalInsights" version="0.10.0-preview" targetFramework="net45" />
```
Comment (markcowl) [inline src/ResourceManager/Common/Commands.ResourceManager.Common/packages.config]: same comment
